### DISCUSSION TITLE
QueryClientProvider in vscode says "Property 'children' does not exist on type"

### DISCUSSION THREAD

**Discussion by mozeryansky** [TanStack/query#3491] [Q&A] [answered]
I started getting this message and I'm not sure where it's from. Everything works fine, but I don't know why the warning exists. Does anyone know what this means?

```
Type '{ children: Element; client: QueryClient; }' is not assignable to type 'IntrinsicAttributes & QueryClientProviderProps'.
  Property 'children' does not exist on type 'IntrinsicAttributes & QueryClientProviderProps'.
  ```
<img width="729" alt="Screen Shot 2022-04-09 at 3 13 43 PM" src="https://user-images.githubusercontent.com/3287719/162590169-ac487830-9e67-49ca-8ed2-ab1991438f20.png">

**Comment by joyfuI** [comment, marked as answer]
https://github.com/tannerlinsley/react-query/issues/3476
